style2.css

```diff
@@ -131,7 +138,258 @@ img {
   margin-top: 10px;
 }
 
+/* grid */
+
+#grid-container {
+  margin: 40px;
+
+  visibility: hidden;
+  opacity: 0;
+
+  background: #ecf0f1;
+  padding: 6px;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+  position: absolute;
+  top: 15%;
+  left: 5px;
+}
+.currency-grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 10px;
+  max-width: 600px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.grid-header {
+  font-weight: bold;
+  background-color: #f0f0f0;
+  padding: 10px;
+}
+
+.grid-item {
+  background-color: #fafafa;
+  padding: 10px;
+  border: 1px solid #ddd;
+}
+.tradingview-widget-container {
+  position: absolute;
+  top: 20%;
+  right: 30px;
+}
+
+.nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #69a6ce;
+  font-size: 24px;
+  font-weight: bold;
+  text-transform: uppercase;
+  text-align: center;
+  padding: 10px 20px;
+}
+
+.nav h1 {
+  margin: 0 auto 0;
+  padding: 10px 20px;
+  color: #d4e6f1;
+  text-shadow: 2px 2px rgba(0, 0, 0, 0.1);
+  font-family: Arial, sans-serif;
+}
+
+/* #yellow-star {
+  color: yellow;
+} */
+
+.bell {
+  display: none;
+  width: 50px;
+  height: 30px;
+  margin: 0 auto;
+  text-align: center;
+  padding: 10px 0;
+  box-sizing: content-box;
+  cursor: pointer;
+}
+.bell-text {
+  visibility: hidden;
+
+  width: 120px;
+  background-color: rgba(0, 0, 0, 0.232);
+  color: white;
+  text-align: center;
+  border-radius: 6px;
+  padding: 5px 0;
+
+  position: relative;
+  z-index: 1;
+  bottom: 65px;
+  right: 95px;
+}
+
+.bell:hover .bell-text {
+  visibility: visible;
+}
+
+.little-text {
+  margin: 0;
+  position: relative;
+  top: -15px;
+  text-align: right;
+  font-size: 6px;
+  box-sizing: border-box;
+}
+
+.market-open {
+  display: flex;
+  width: fit-content;
+  position: relative;
+  top: -20px;
+  right: -450px;
+  text-align: center;
+  align-items: center;
+}
+#saved-alerts-container {
+  width: 200px;
+  height: 40px;
+  position: absolute;
+  top: 14%;
+  left: 70px;
+  z-index: 1;
+}
+#alerts-container {
+  position: absolute;
+  top: 10%;
+  right: 150px;
```

Comment:
Have you tested this on smaller screens? (phone or tablet)

Reply:
yes, but i will re-check it now